src/pybind/mgr/dashboard/services/auth.py

```diff
@@ -277,3 +288,64 @@ def _check_authorization(self, username):
         if not AuthManager.authorize(username, sec_scope, sec_perms):
             raise cherrypy.HTTPError(403, "You don't have permissions to "
                                           "access that resource")
+
+class Oauth2JWTManager(object):
```

Comment:
Redundant since 2.7:
```suggestion
class Oauth2JWTManager:
```